lib/vanagon/platform/windows.rb

```diff
@@ -34,6 +37,17 @@ def package_name(project)
         "#{project.name}-#{project.version}-#{project.release}-#{@architecture}.msi"
       end
 
+      def add_repository(definition)
```

Comment:
Having some yard doc here would help to know what definition is expected to be (always a url, or something else).